earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +50,16 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+env.Command(
```

Comment:
This may be because I don't really like the syntax for the env.Command in the first place. The problem is made somewhat worse because scons doesn't have thorough documentation for it. Also, I don't know if env.copyfile will return a "Command" that the env.Command function can use. I think it just does the copy instead, but I could be wrong. Scons is a bit complex still in my mind. :)